### PR TITLE
Bugfix obstacle deletion

### PR DESCRIPTION
**Pull request by marcelkuipers**
I fixed a bug where obstacles wouldn't be deleted after some refactoring had been done.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/obstacle/ObstacleSpawner.java [hunk 1]
```diff
@@ -127,4 +130,10 @@ public AbstractObstacleFactory getObstacleFactory() {
     private float getBound(BoundingBox boundingBox) {
         return boundingBox.getCenter().z;
     }
+
+    public LinkedList<Obstacle> deleteObstacles() {
+        LinkedList<Obstacle> temp = (LinkedList<Obstacle>) deleteList.clone();
+        deleteList = new LinkedList<Obstacle>();
+        return temp;
+    }
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/obstacle/ObstacleSpawner.java:142]: Missing javadoc

**Comment by NilsHullegien** [comment]
Besides my comment, the code looks good!

**Comment by marcelkuipers** [comment]
Oops, fixed it now.

**Comment by NilsHullegien** [comment]
Looks good now, I approve!

**Comment by dVoorhout** [comment]
looks good, I noticed no slowdown after 5 mins, will merge

**Pull request merged**